.github/workflows/erlang.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Erlang CI
+
+"on": [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        otp: ['24', '25', '26']
+        rebar: ['3.22']
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: erlef/setup-beam@v1
+        id: setup-beam
+        with:
+          otp-version: ${{matrix.otp}}
+          rebar3-version: ${{matrix.rebar}}
+      - name: Restore _build
+        uses: actions/cache@v3
+        with:
+          path: _build
+          key: "_build-cache-for\
+                -os-${{runner.os}}\
+                -otp-${{steps.setup-beam.outputs.otp-version}}\
+                -rebar3-${{steps.setup-beam.outputs.rebar3-version}}\
+                -hash-${{hashFiles('rebar.lock')}}"
+      - name: Restore rebar3's cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/rebar3
+          key: "rebar3-cache-for\
+                -os-${{runner.os}}\
+                -otp-${{steps.setup-beam.outputs.otp-version}}\
+                -rebar3-${{steps.setup-beam.outputs.rebar3-version}}\
+                -hash-${{hashFiles('rebar.lock')}}"
+      - name: Compile
+        run: rebar3 compile
+      - name: Format check
+        run: rebar3 format --verify
+      - name: Run tests and verifications
+        run: rebar3 test
+      - name: Run meta verifications
+        run: rebar3 as test test
```

Comment:
Notice this portion, that effectively replaces the meta SUITE. There's a comment on 93d7429 as to why this was the chosen solution.